### DISCUSSION TITLE
Download failed

### DISCUSSION THREAD

**Discussion by Pit1288** [aclap-dev/video-downloadhelper#755] [Help and Q&A]
### Your question

I must add that I've downloaded another video correctly from the same page.

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "https://screening.globo.com/content/helenas-shadow/",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [tls @ 0000018f9a58d140] IO error: End of file\r\n[https @ 0000018f9a58aa00] Stream ends prematurely at 2030766, should be 2433848\r\n[tls @ 0000018f9a58d140] IO error: End of file\r\n[aac_adtstoasc @ 0000018f9a53c700] Error parsing ADTS frame header!\r\n[mp4 @ 0000018f9b2e0b80] Error applying bitstream filters to an output packet for stream #1: Invalid data found when processing input\r\n[aost#0:1/copy @ 0000018f9aaeec80] Error submitting a packet to the muxer: Invalid data found when processing input\r\n[out#0/mp4 @ 0000018f9a590100] Error muxing a packet\r\n",
    "id": "coapp_failure_7ac8b9b6-a990-40d1-a072-2cde792f89fe",
    "downloadable_id": "downloadable_7541137512877121"
  }
}

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"D:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "http_media_download_strategy": "inbrowser",
    "download_directory": "C:\\Users\\Piter\\Videos",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": false,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_wide_ui": true,
    "open_count_store": 10
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0.1 64-bit, licensed VDH 9.0.2.3a1 beta, CoApp 2.0.19.

I visited your web page & it was offering both a trailer & a complete movie.  When I clicked the link for the movie, it said I needed to log in.  I'm not going to register for this site.

When I clicked the link for the trailer, & then launched playback of the trailer for a couple of seconds, I got this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/9ad46d4d-345e-48aa-b700-e721f35564fe)

VDH downloaded the 720p variant without problem.  It played fine in VLC.

Maybe this doesn't prove anything because a 3-minute trailer is nothing like a movie of an hour or two.  But I was able to get this much, at least.

There was a FIrefox update earlier today.  Also, I am running the latest VDH beta & you are not.  Those are 2 possible avenues of investigation you might look into.

**Reply by GrampaWildWilly** [reply]
Forgot something.  We have been seeing these ADTS errors lately.  These usually indicate transient errors in the data stream.  Retrying often helps.  Unfortunately, retrying perhaps more than 5 times, more than 10 times, is what it takes to finally make the web site give you something that you can download.  These problems appear to be in the web sites, not VDH.  Perhaps you are encountering one of these cases.
